kbase-extension/static/kbase/js/widgets/narrative_core/jobCommChannel.js

```diff
@@ -74,6 +104,35 @@ define([
             this.runtime = Runtime.make();
             this.jobStates = {};
             this.handleBusMessages();
+            this.debug = devMode
+                ? (...args) => {
+                      // eslint-disable-next-line no-console
+                      console.log(...args);
+                  }
+                : () => {
+                      /* no op */
+                  };
+        }
```

Comment:
added in a debug function that logs messages if `devMode` is true and does nothing otherwise.